README.md

```diff
@@ -10,4 +10,16 @@ A simple Node.js app using restify that interacts with the googleMaps directions
 
 ## Swagger 2.0 docs
 
+This service currently provides the following endpoints:
+
+* POST to api/v1/diretions with a body :
+
+```js
+{
+	"origin": "string value",
+	"destination": "string value",
+	"departure_time": "time of departure in epoch"
```

Comment:
Don't put underscores in JSON keys, the cause issues with parsers like Jackson.

Reply:
So, what is ideal here. camelCase?